README.md

```diff
@@ -12,6 +12,9 @@ please see [the documentation](https://docs.dissect.tools/en/latest/projects/dis
 pip install dissect.target
 ```
 
+If you wish to use the YARA plugin (`target-query -f yara`), you can install `dissect.target[yara]` to automatically 
+install the `yara-python` dependency.
+
 This module is also automatically installed if you install the `dissect` package.
```

Comment:
```suggestion
This module is also automatically installed if you install the `dissect` package.

If you wish to use the YARA plugin (`target-query -f yara`), you can install `dissect.target[yara]` to automatically 
install the `yara-python` dependency.
```

I changed my mind :smile: with my previous suggestion it's unclear what "This module" refers to, so better to put this new sentence below.